src/libYARP_OS/src/Node.cpp

```diff
@@ -9,9 +9,12 @@
 #include <yarp/conf/compiler.h>
 #include <yarp/os/Node.h>
 #include <yarp/os/Mutex.h>
+#include <yarp/os/LogStream.h>
```

Comment:
Please do not use LogStream in YARP_OS port related classes. Since log generated by yDebug etc. can be forwarded to the logger using a port, it is better for now to avoid using them around these classes, as this might cause recursive loops...
Unfortunately we have a lots of logging methods around the YARP code... One day we will make disappear them all and leave only one, but for now have a look at what is already used in the class and use the same...